IPython/testing/iptest.py

```diff
@@ -58,6 +59,24 @@
 warnings.filterwarnings('ignore', 'wxPython/wxWidgets release number mismatch',
                         UserWarning)
 
+# Enable printing all warnings raise by IPython's modules
+warnings.filterwarnings('default', message='.*', category=Warning, module='IPy.*')
+
+
+if version_info < (6,):
```

Comment:
I think a minor revision bumping a fully backward-compatible minor revision of a dependency is fine. I'm not sure if we will, but I don't think that bumping required traitlets to 4.1 is the sort of change that forces 5.0 (we may well be preparing to release 5.0 by the time we get to it anyway, so it may be moot). There's certainly *no* reason that we would need wait *2* major revisions before bumping a minor revision of a dependency.